.github/workflows/module_integration.yml

```
@@ -145,7 +145,7 @@ jobs:
           pip install 'pip>=23.0,<23.1'
           pip install -U setuptools
           pip install -r requirements.txt
-          pip install deprecated 'pyarrow!=12.0.0' tables geopandas numpy
+          pip install deprecated 'pyarrow!=12.*' tables geopandas numpy
```

```suggestion
          pip install deprecated 'pyarrow==11.0.0' tables geopandas numpy
```
I suggest pinning a known-working version, rather than trying to chase exclusions of presently non-working versions. Then we can stick an issue in the backlog to revisit the selected version sometime later.

---

Yeah especially now that I think it _should_ be working in 12.x, I was going to come back and do this instead.

---

As a side note I'm never again going to put `deprecated` as the first package after `pip install` because if you don't know that's a package it really looks like part of the pip command.

---

I'll admit that threw me off a bit when I saw it.